.github/workflows/push.yml

```
@@ -0,0 +1,63 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11
```

if you can maybe use alpine version? its almost half the size, might save a few seconds on builds 🤷 


```suggestion
        image: postgres:11-alpine
```